patches/chrome-installer-mini_installer-BUILD.gn.patch

```diff
@@ -11,6 +11,15 @@ index c365088b90b7a91faf52f5b108f64621a219b11f..3be0b71dfb7c8694da3d30608a3be627
      "configuration.cc",
      "configuration.h",
      "decompress.cc",
+@@ -37,7 +37,7 @@ source_set("lib") {
+     "mini_installer.cc",
+     "mini_installer.h",
+     "mini_installer.rc",
+-    "mini_installer_constants.cc",
++    "//brave/installer/mini_installer/mini_installer_constants.cc",
```

Comment:
if we are replacing the entire file it's better to use `chromium_src` which will work automatically

Reply:
Ok, it would be good to move `//brave/installer/mini_installer/mini_installer_constants.cc` into `chromium_src`.